docs/troubleshooting/monitoring.md

```diff
@@ -316,3 +316,255 @@ You may encounter this when you install the Harvester v1.3.0 or higher version c
+    meta.helm.sh/release-namespace: cattle-monitoring-system
+  creationTimestamp: "2023-01-09T11:04:33Z"
+  labels:
+    app: rancher-monitoring-crd-manager
+    app.kubernetes.io/managed-by: Helm
+  name: rancher-monitoring-crd-manager
+  ...
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rancher-monitoring-crd-manager
+subjects:
+- kind: ServiceAccount
+  name: rancher-monitoring-crd-manager
+  namespace: cattle-monitoring-system
+
+$ kubectl get configmap -n cattle-monitoring-system rancher-monitoring-crd-manifest
+apiVersion: v1
+data:
+  crd-manifest.tgz.b64: ...
+kind: ConfigMap
+metadata:
+  annotations:
+    meta.helm.sh/release-name: rancher-monitoring-crd
+    meta.helm.sh/release-namespace: cattle-monitoring-system
+  creationTimestamp: "2023-01-09T11:04:33Z"
+  labels:
+    app.kubernetes.io/managed-by: Helm
+  name: rancher-monitoring-crd-manifest
+  namespace: cattle-monitoring-system
+  ...
+
+$ kubectl get ServiceAccount -n cattle-monitoring-system rancher-monitoring-crd-manager
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    meta.helm.sh/release-name: rancher-monitoring-crd
+    meta.helm.sh/release-namespace: cattle-monitoring-system
+  creationTimestamp: "2023-01-09T11:04:33Z"
+  labels:
+    app: rancher-monitoring-crd-manager
+    app.kubernetes.io/managed-by: Helm
+  name: rancher-monitoring-crd-manager
+  namespace: cattle-monitoring-system
+  ...
+```
+
+### Root Cause
+
+Those objects do not have the related labels/annotations required by the `ManagedChart` object, hence they are marked as `missging`.
+
+### Workaround
+
+To fix the issue, you may follow the steps below to re-create the `rancher-monitoring-crd` Managedchart object.
+
+1. Confirm the chart version of `rancher-monitoring-crd` from the `harvester-cluster-repo` service.
+
+    Get the `harvester-cluster-repo` service IP.
+
+    ```
+    $ kubectl get service -A | grep repo
+    cattle-system                     harvester-cluster-repo                        ClusterIP      10.53.146.66    <none>            80/TCP                         8d
+
+    The IP address is 10.53.146.66.
+    ```
+
+    Get the chart version from the `index.yaml` in the `harvester-cluster-repo`.
+
+    ```
+    $ curl http://10.53.146.66/charts/index.yaml | grep "rancher-monitoring"
+    ...
+    name: rancher-monitoring
+        - rancher-monitoring-102.0.0+up40.1.2.tgz
+      rancher-monitoring-crd:
+          catalog.cattle.io/release-name: rancher-monitoring-crd
+        description: Installs the CRDs for rancher-monitoring.
+        name: rancher-monitoring-crd
+        - rancher-monitoring-crd-102.0.0+up40.1.2.tgz
+
+    The chart version is 102.0.0+up40.1.2.
+    ```
+
+    The chart version should be the same as the `version` field in the `rancher-monitoring-crd` ManagedChart object.
+
+    :::note
+
+    If those two versions are different, you need to stop and ask help from Harvester support.
```

Comment:
Should all users be encouraged to contact support directly? If so, how exactly are they supposed to do that?

cc: @bk201

Reply:
Only paid customers can contact Harvester support directly. Community users will ask on github or slack channel.

Reply:
Enterprise users know how to report issues and contact SUSE support. A better approach here might be to create a page like "How to ask for help?" that describes how to get help from Slack/Github. And link to that page. Although I think the work can be done in another PR.

Reply:
Added.